Rewrite tutorial to use the scaffold tool (#204)
cosmos/sdk-tutorials: nameservice/tutorial/00-intro.md

- A working [`$GOPATH`](https://github.com/golang/go/wiki/SettingGOPATH)
- Github account and either [Github CLI](https://hub.github.com/) or [Github Desktop (64-bit required)](https://help.github.com/en/desktop/getting-started-with-github-desktop/installing-github-desktop)
- Desire to create your own blockchain!

> - The [scaffold tool](https://github.com/cosmos/scaffold) will be used to go through this tutorial. If you go to the repo then following the listed instructions will enable you to use the tool. 

Author: ```suggestion
- The [scaffold tool](https://github.com/cosmos/scaffold) will be used to go through this tutorial. If you go to the repo and follow the installation instructions you will be able to use the tool. 
```

> - The [scaffold tool](https://github.com/cosmos/scaffold) will be used to go through this tutorial. If you go to the repo and follow the installation instructions you will be able to use the tool. 

Reviewer: ```suggestion
- The [scaffold tool](https://github.com/cosmos/scaffold) will be used to go through this tutorial. Clone the repo and install with `git clone git@github.com:cosmos/scaffold.git && cd scaffold && make`. Check out the repo for more detailed instructions.
```
